src/handlers/comment/handlers/index.ts

```diff
@@ -31,7 +31,7 @@ export const commentParser = (body: string): IssueCommentCommands[] => {
   // TODO: As a starting point, it may be simple but there could be cases for the comment to includes one or more commands
   // We need to continuously improve to parse even complex comments. Right now, we implement it simply.
   const commandList = Object.values(IssueCommentCommands) as string[];
-  const result = commandList.filter((command: string) => body.includes(command));
+  const result = commandList.filter((command: string) => (command === "/multiplier" ? body.startsWith(command) : body.includes(command)));
```

Comment:
Isn't it better to implement this update for all commands? 
```
const result = commandList.filter((command: string) => body.startsWith(command));
```

Reply:
As you can see the comments, there could be cases for the comment to includes one or more commands.

Reply:
right now we don't support multiple commands in a single comment

I'm trying to understand why we should have a special treat **only** for the `/multiplier` command

As far as I understand it is simpler to use `body.startsWith(command)`

@0xcodercrane help

Reply:
We don't support multiple commands in a single comment that's right. In terms of hardcoded `/multiplier`, I agree to remove it from the line. If there's a case to use it for comparison, you should use it like [IssueCommentsCommands.MULTIPLIER](https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/comment/commands.ts#L7)